### DISCUSSION TITLE
[Json] What am I wrong?

### DISCUSSION THREAD

**Discussion by shtse8** [prisma/prisma#14675] [Q&A] [answered]
<img width="507" alt="image" src="https://user-images.githubusercontent.com/8020099/183151035-31fc3c0e-e0f8-49ea-80d4-e0cc5e8696da.png">

```
model Article {
  id             Int           @id @default(autoincrement())
  data           ArticleData?
}

model ArticleData {
  articleId Int     @id
  article   Article @relation(fields: [articleId], references: [id])
  words     Json
}
```

I can't `where` the relation json field. What am I doing wrong?

--------------
From official doc:

https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields
<img width="942" alt="image" src="https://user-images.githubusercontent.com/8020099/183151382-6234c3f1-9d1c-4ab7-851d-6cd143405a9a.png">

**Comment by shtse8** [comment]
It works well with
```ts
const article = await prisma.article.findFirst({
    where: {
      OR: [
        { data: null },
        {
          data: {
            words: {
              path: '$',
              equals: Prisma.AnyNull,
            },
          },
        },
      ],
    },
    orderBy: {
      id: 'asc',
    },
  })
```

But I don't figure out why the above isn't work.

**Comment by pantharshit00** [comment, marked as answer]
Looks like example in the docs is incorrect. Thanks for flagging this. I have created this docs issue to resolve this: https://github.com/prisma/docs/issues/3845

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
